web-apps/genesys-spark-examples/src/utils/generateFontAwesomeIcons.js

```diff
@@ -1,197 +1,111 @@
 const icons = [
```

Comment:
This file will not have this change when I merge the PR (It will have a different one) but I will leave it like this for now to simplify the review process